tests/integration/targets/locale_gen/tasks/non_debian.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: "Non dnf RedHat systemd are not supported"
+  ansible.builtin.meta: "end_play"
+  when:
+    - "ansible_pkg_mgr == 'yum'"
+
+- name: "Prepare locale packages"
+  block:
+    - name: "Install locale packages for RedHat"
+      ansible.builtin.dnf:
+        name:
+          - "glibc-langpack-pl"
+          - "glibc-langpack-en"
+        state: "present"
+        update_cache: "yes"
+      when:
+        - "ansible_os_family == 'RedHat'"
+    - name: "Install locale packages for Archlinux"
+      community.general.pacman:
+        name: "glibc-locales"
```

Comment:
The requirement to install these packages should be mentioned in the module documentation, for the benefit of new users.

Reply:
This documented in 

* https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/configuring_basic_system_settings/assembly_changing-basic-environment-settings_configuring-basic-system-settings#proc_configuring-the-system-locale_assembly_changing-basic-environment-settings

* https://wiki.archlinux.org/title/Locale


Feel free to add the inline commit, if this links definitely should be present in module